knx/dpt/types_17.go

```diff
@@ -0,0 +1,38 @@
+package dpt
+
+import "fmt"
+
+// DPT_17001 represents DPT 17.001 / Scene Number.
+type DPT_17001 uint8
+
+func (d DPT_17001) Pack() []byte {
+	if d > 63 {
+		return packU8(63)
+	} else {
+		return packU8(uint8(d))
+	}
+}
+
+func (d *DPT_17001) Unpack(data []byte) error {
+	var value uint8
+
+	if err := unpackU8(data, &value); err != nil {
+		return err
+	}
+
+	if *d <= 63 {
```

Comment:
This must compare against `value` not `d`.

Reply:
I see you already fixed this on master, I will bring that change to the PR later. :)